cli/install/install.go

```diff
@@ -1109,6 +1114,88 @@ func installTarantoolEE(binDir string, includeDir string, installCtx InstallCtx,
 	return nil
 }
 
+// installTarantoolDev installs tarantool from the local build directory.
+func installTarantoolDev(ttBinDir string, ttIncludeDir, buildDir,
+	includeDir string) error {
+	var err error
+
+	// Validate build directory.
+	if buildDir, err = filepath.Abs(buildDir); err != nil {
+		return fmt.Errorf("failed to get absolute path: %v", err)
+	}
+	if !util.IsDir(buildDir) {
+		return fmt.Errorf("directory %v doesn't exist, or isn't directory", buildDir)
+	}
+
+	defaultIncPath := filepath.Join(buildDir, "tarantool-prefix", "include", "tarantool")
+	if includeDir != "" {
+		// Validate headers directory.
+		if includeDir, err = filepath.Abs(includeDir); err != nil {
+			return fmt.Errorf("failed to get absolute path: %v", err)
+		}
+		if !util.IsDir(includeDir) {
+			return fmt.Errorf("directory %v doesn't exist, "+
+				"or isn't directory", includeDir)
+		}
+	} else {
+		// Check for default.
+		if util.IsDir(defaultIncPath) {
+			log.Infof("tarantool headers directory set as %v", defaultIncPath)
+			includeDir = defaultIncPath
+		}
+	}
+
+	if includeDir == "" {
+		log.Warn("Tarantool headers directory wasn't specified and" +
```

Comment:
need to add space between “and” && “was”

Reply:
also u use "wasn't" and then "was not"
i think it would be prettier to use one variant

Reply:
Fixed.